swagger2.0/oic.wk.d.swagger.json

```diff
@@ -2,71 +2,81 @@
-    }
+      "name": "OCF Data Model License",
+      "url": "https://openconnectivityfoundation.github.io/core/LICENSE.md",
+      "x-copyright": "Copyright 2016-2019 Open Connectivity Foundation, Inc. All rights reserved."
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
   },
-  "schemes": ["http"],
-  "consumes": ["application/json"],
-  "produces": ["application/json"],
+  "schemes": [
+    "http"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
   "paths": {
     "/oic/d" : {
       "get": {
         "description": "Known Resource that is hosted by every Server.\nAllows for logical Device specific information to be discovered.\n",
         "parameters": [
-          {"$ref": "#/parameters/interface"}
+          {
+            "$ref": "#/parameters/interface"
+          }
         ],
         "responses": {
-            "200": {
-              "description" : "",
-              "x-example":
-                {
-                  "n":    "Device 1",
-                  "rt":   ["oic.wk.d"],
-                  "di":   "54919CA5-4101-4AE4-595B-353C51AA983C",
-                  "icv":  "ocf.1.0.0",
-                  "dmv":  "ocf.res.1.0.0, ocf.sh.1.0.0",
-                  "piid": "6F0AAC04-2BB0-468D-B57C-16570A26AE48"
-                },
-              "schema": { "$ref": "#/definitions/Device" }
+          "200": {
+            "description": "",
+            "x-example":
+              {
+                "n":    "Device 1",
+                "rt":   ["oic.wk.d"],
+                "di":   "54919CA5-4101-4AE4-595B-353C51AA983C",
+                "icv":  "ocf.1.0.0",
+                "dmv":  "ocf.res.1.0.0, ocf.sh.1.0.0",
+                "piid": "6F0AAC04-2BB0-468D-B57C-16570A26AE48"
+              },
+              "schema": {
+                 "$ref": "#/definitions/Device"
+               }
             }
-        }
+         }
       }
     }
   },
   "parameters": {
     "interface" : {
-      "in" : "query",
-      "name" : "if",
-      "type" : "string",
-      "enum" : ["oic.if.r", "oic.if.baseline"]
+      "in": "query",
+      "name": "if",
+      "type": "string",
+      "enum": ["oic.if.r", "oic.if.baseline"]
     }
   },
   "definitions": {
-    "Device" : {
+    "Device": {
       "properties": {
-        "rt" :  {
+        "rt":  {
           "description": "Resource Type of the Resource",
           "items": {
-            "maxLength": 64,
+            "enum": ["oic.wk.d"],
             "type": "string"
           },
           "minItems": 1,
           "readOnly": true,
           "type": "array"
         },
-        "ld" :  {
+        "ld":  {
```

Comment:
I don't recall this Property but I see it in the spec.  Is it intended to mirror the value of "mnpn" from the oic.wk.con.p Resource?  That won't affect this PR but it's not defined anywhere and if they are tied, it might be useful to include that in the description of either "ld" or "mnpn".